src/VisualStudio/Core/Def/LanguageService/AbstractPackage`2.cs

```diff
@@ -36,7 +36,7 @@ protected AbstractPackage()
 
     protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
     {
-        await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
```

Comment:
>true [](http://example.com/codeflow?start=79&length=4)

no need to capture the context here as the next line will move us back to the UI thread